platform/web/js/libs/library_godot_audio.js

```diff
@@ -1252,15 +1323,14 @@ const _GodotAudio = {
 			startOptions
 		) {
 			GodotAudio.SampleNode.stopSampleNode(playbackObjectId);
-			const sampleNode = GodotAudio.SampleNode.create(
+			GodotAudio.SampleNode.create(
 				{
 					busIndex,
 					id: playbackObjectId,
 					streamObjectId,
 				},
 				startOptions
 			);
```

Comment:
Why the change here?

Reply:
before it was creating a const sampleNode, and then calling .start() . I didnt want it to start the node before the position worklet had been connected.  Just to ensure it wasnt getting out of sync